src/bootstrap/Cargo.toml

```diff
@@ -80,6 +82,3 @@ build-metrics = ["sysinfo"]
 # dependencies, only bootstrap itself.
 [profile.dev]
 debug = 0
-[profile.dev.package]
-# Only use debuginfo=1 to further reduce compile times.
-bootstrap.debug = 1
```

Comment:
Why did you remove this? Removing this disables debuginfo for bootstrap itself.

Reply:
...I don't recall doing it :sweat_smile: Reverted